src/ts/parsers/page.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { buildSpecFrom } from "./page";
```

Comment:
I think this file belongs in a different directory (a test folder, or at least a renaming to `page.test.ts` -- unless the convention in extension testing is different).

Reply:
I think the `filename.spec.ts` naming is quite well-established, and I thought I would keep the tests next to their source files like many people like to do to avoid a separate testing directory in this project.

Reply:
Apparently I've used already `.test.ts[x]` in React component tests so changed the name to have it consistent. I still very much like the idea that the `.spec` file defines what the module does, though, and those are automatically discovered by Jest just like `.test` files.

Reply:
Ah! It's an angular convention. Did not realize that one either. You learn something new every day :)